auraed/src/cells/cell_service/executables/executable_name.rs

```diff
@@ -6,6 +6,13 @@ use validation::{ValidatedField, ValidationError};
 #[derive(Debug, Clone, Ord, PartialOrd, Eq, PartialEq, Hash)]
 pub struct ExecutableName(String);
 
+#[cfg(test)]
+impl ExecutableName {
+    pub(crate) fn new(name: String) -> Self {
```

Comment:
I think this should have the `#[cfg(test)]` attribute.

Reply:
it doesn't inherit from the impl?

Reply:
I think you are right. It should inherit; I missed that it was above. I do prefer it on the fn though (but wouldn't have not approved had I seen it).